pyaerocom/io/read_ghost.py

```diff
@@ -178,7 +178,7 @@ class ReadGhost(ReadUngriddedBase):
     #: Default flags used to invalidate data points (these may be either from
     #: provided flag or qa variable, or both, currently only from qa variable)
     DEFAULT_FLAGS_INVALID = {'qa' : np.asarray([0, 1, 2, 3, 6, 8, 9, 10, 12, 
-                                                13, 14, 17, 18, 22, 25, 30, 40, 
+                                                13, 14, 15, 18, 22, 25, 30, 40, 
```

Comment:
I guess this is for a reason (removing 17 and using 15 instead)? :)

Reply:
From Dene's e-mail today:The default codes to remove are now: [0, 1, 2, 3, 6, 8, 9, 10, 12, 13, 14, 15, 18, 19, 22, 25, 30, 40, 41, 42]